### ISSUE TITLE
WCAG Accessibility Support for DatePicker

### ISSUE THREAD

**Issue by snuser390** [uxsolutions/bootstrap-datepicker#2440]
### Expected behaviour
DatePicker should be accessible through keyboard navigation. Assistive technologies( like Voicover, JAWS) should be able to read the dates in the calendar.

### Actual behaviour
The previous and next buttons are not accessible through keyboard. The dates in the calendar are not announced by assistive technologies.

### Datepicker version used
Version 2.0
ex. 1.6.1.

### Example code

Jsfiddle example to reproduce the problem.

**Comment by heatworld1** [comment]
Yea I agree and would make my life easier than removing this plugin and searching for another one.

**Comment by perriard** [comment]
Any news on this one? :-)
